cohttp-lwt-unix/src/server.mli

```diff
@@ -4,6 +4,7 @@
 include Cohttp_lwt.S.Server with module IO = Io
 
 val resolve_file : docroot:string -> uri:Uri.t -> string
+[@@@ocaml.alert deprecated "Please use Cohttp.Server_utils.resolve_local_file."]
```

Comment:
Let's hold off on the deprecation warnings due to the implications of this fix.  We want people to upgrade quickly -- the deprecations can be added in a future release.

Reply:
Done.